content/usage/_index.md

```diff
@@ -93,28 +93,25 @@ This command will open configmap specs in vim editor. Do the required changes, s
 
 #### Disable notifications
 
-If you want to stop receiving notifications from BotKube, run
-**@BotKube notifier stop**
-from the configured channel where BotKube is added. You will no longer receive notifications from the BotKube
+If you want to stop receiving notifications from BotKube, run **@BotKube notifier stop** from the configured channel where BotKube is added. You will no longer receive notifications from the BotKube in a given communication platform.
 
-![notifier_stop](/images/notifier_stop.png)
-![notifier_stop](/images/mm_notifier_stop.png)
+![notifier-stop](/images/notifier-stop.png)

```

Comment:
The only thing that I would change is the username. It would become quite messy if all of us will add "personalized" screens. If I will change e.g. only one of the notification messages (only for stop) and the update the screen, it would look strange. 

We can start updating them in the consistent way. Same as a did for our main README.md. It can be anything, for now it's "Ricky" inspired user. 
![Screen Shot 2022-07-29 at 13 47 37](https://user-images.githubusercontent.com/17568639/181752709-8e260416-7fba-4d4e-a703-037bd9f7568a.png)
![Screen Shot 2022-07-29 at 13 47 25](https://user-images.githubusercontent.com/17568639/181752718-0f5f9a3b-f2e7-4a42-b18c-eafc12efb584.png)
![Screen Shot 2022-07-29 at 13 47 09](https://user-images.githubusercontent.com/17568639/181752721-35f9e0c4-9922-4a3b-a5d1-9d3d000ea1e8.png)

I'm open to anything but let's be "generic" and consistent 👍